src/ExposureContext.tsx

```diff
@@ -73,12 +77,20 @@ const ExposureProvider: FunctionComponent = ({ children }) => {
     })
   }, [getLastDetectionDate])
 
-  const getCurrentExposures = useCallback(() => {
-    const cb = (exposureInfo: ExposureInfo) => {
-      setExposureInfo(exposureInfo)
+  useEffect(() => {
+    const handleAppStateChange = async () => {
+      const exposureInfos = await getCurrentExposures()
```

Comment:
I think the name tripped me because I forgot that `ExposureInfo` is a collection, maybe `exposureInfoData` would be ok?